### PR TITLE
supress or disable edit bar done

### LINKED ISSUE #64: Support IsEditable & Disable Edit Bar
Currently there is no way to supress or disable edit bar in DNN; therefore, IsEditable functionality is temporarily removed and causes issues with 3rd party modules. 

Beginning with DNN 9.8, there will be an option to disable Edit Bar. Vanjaro should be able to use this option to support IsEditable with 3rd party modules.

### PR DESCRIPTION
**Pull request by mksmanish79**
close #64

### CODE REVIEW

File: Portals/_default/Skins/Vanjaro/Base.ascx.cs [hunk 1]
```diff
@@ -103,8 +103,8 @@ protected override void OnInit(EventArgs e)
         private void HandleEditMode()
         {
             ///Forcing View Mode Until we can find a solution to prevent Edit Bar from appearing. 
-            ToggleUserMode("VIEW");
-            return;
+            //ToggleUserMode("VIEW");
+            //return;
```
Comment (meetmandeep) [inline Portals/_default/Skins/Vanjaro/Base.ascx.cs]: Please remove these commented lines. They're not needed.

**Review by meetmandeep — CHANGES_REQUESTED** [review]

File: Portals/_default/Skins/Vanjaro/Base.ascx.cs (hunk 1 above)
Comment (mksmanish79) [reply Portals/_default/Skins/Vanjaro/Base.ascx.cs]: done

File: DesktopModules/Vanjaro/Core/Library/Managers/SettingManager.cs [hunk 2]
```diff
@@ -444,6 +444,10 @@ private static void DoIUpgradeable(PortalInfo pinfo)
                     UpdatePortalSettings(SeoSettings, pinfo.PortalID, uInfo.UserID);
                     #endregion
                 }
+                else
+                {
+                    HostController.Instance.Update("DisableEditBar", "False");
+                }
```
Comment (meetmandeep) [inline DesktopModules/Vanjaro/Core/Library/Managers/SettingManager.cs]: This needs to be updated to support upgrades. It should happen in IUpgradeable targeting a specific version.

**Review by meetmandeep — CHANGES_REQUESTED** [review]

File: DesktopModules/Vanjaro/Core/Library/Managers/SettingManager.cs (hunk 2 above)
Comment (mksmanish79) [reply DesktopModules/Vanjaro/Core/Library/Managers/SettingManager.cs]: changes updated

**Review by meetmandeep — APPROVED** [review]

**Pull request merged**